src/nvim/os/os_defs.h

```diff
@@ -14,13 +13,6 @@
 # include "nvim/os/unix_defs.h"
 #endif
 
-/// File descriptor number used for standard IO streams
-enum {
-  OS_STDIN_FILENO = STDIN_FILENO,
-  OS_STDOUT_FILENO = STDOUT_FILENO,
-  OS_STDERR_FILENO = STDERR_FILENO,
-};
```

Comment:
@ZyX-I any objection to removing this?

Reply:
This is not used in unit tests currently, so no objections. If it was then `enum` would be required.